templates/advanced/package.yaml

```diff
@@ -4,15 +4,18 @@ name: Advanced Template Package
 # Required: A description of what's in the action package.
 description: Action package description
 
-# Required: A link to where the documentation on the package lives.
-documentation: https://github.com/...
+# Required: Point to main documentation markdown file for the package.
+documentation: README.md
+
+# Package version number, recommend using semver.org
+version: 0.0.1
 
 dependencies:
   conda-forge:
   - python=3.10.14
   - uv=0.2.4
```

Comment:
I saw some variations of `uv=0.2.6` here, while the latest is `uv=0.2.9`. What's the rationale when picking the right version and why would **0.2.4** be better than the latest one which is **0.2.9**?

Reply:
Bi-weekly update cadence needs to be set.. so not everyone updating to the latest "whenever" but a set time when we update and test all to a specific uv version.
We cannot have 12 different environment contents, we rather need more of a "golden env." that all of these try to use.